src/__tests__/__snapshots__/rockets.test.js.snap

```diff
@@ -0,0 +1,42 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`A tests for my Rockets component Check whether Rockets component renders 1`] = `
```

Comment:
Hello @AckonSamuel 

- [ ] I believe this should be a test for Missions and not rockets